packages/react-native-cli/src/commands/AutomateSymbolicationCommand.ts

```diff
@@ -79,23 +59,33 @@ export default async function run (projectRoot: string, urls: OnPremiseUrls): Pr
         }
       }
 
-      const { packageJsonIntegration } = await prompts({
-        type: 'confirm',
-        name: 'packageJsonIntegration',
-        message: 'Do you want to add an NPM task to your package.json that you can run to upload Android source maps?',
-        initial: true
-      }, { onCancel })
-
-      if (packageJsonIntegration) {
-        await writeToPackageJson(join(projectRoot, 'package.json'), urls[UrlType.UPLOAD], urls[UrlType.BUILD])
-      }
+      await writeToPackageJson(join(projectRoot, 'package.json'), urls[UrlType.UPLOAD], urls[UrlType.BUILD])
 
       await prompts({
         type: 'text',
         name: 'bugsnagCliInstructions',
         message: BUGSNAG_CLI_INSTRUCTIONS,
         initial: 'Hit enter to continue …'
       }, { onCancel })
+
+      const { iosIntegration } = await prompts({
+        type: 'confirm',
+        name: 'iosIntegration',
+        message: 'Do you want to automatically upload JavaScript source maps as part of the Xcode build?',
+        initial: true
+      }, { onCancel })
+
+      if (iosIntegration) {
+        logger.info('Modifying the Xcode project')
+        await updateXcodeProject(projectRoot, urls[UrlType.UPLOAD], reactNativeVersion as string, logger)
+
+        await prompts({
+          type: 'text',
+          name: 'dsymUploadInstructions',
+          message: DSYM_INSTRUCTIONS,
+          initial: 'Hit enter to continue …'
+        }, { onCancel })
+      }
```

Comment:
I think we agreed that we only want to keep the part that enables source maps and we're not going to add the call to upload to the build - to keep it consistent and understandable.

We probably still want a prompt here that we're about to modify the Xcode project, but not for "automatic upload" and this functionality can be taken out of the Xcode file.

Reply:
Bundled as part of this is setting where the sourcemaps are exported (our happy path) did we want a prompt for that instead or just update with out asking?